build_container/build_container_windows.ps1

```diff
@@ -102,38 +102,38 @@ AddToPath (Resolve-Path "C:\Program Files (x86)\Microsoft Visual Studio\2019\Bui
 
 # CMake to ensure a 64-bit build of the tool (VS BuildTools ships a 32-bit build)
 DownloadAndCheck $env:TEMP\cmake.msi `
-                 https://github.com/Kitware/CMake/releases/download/v3.18.0/cmake-3.18.0-win64-x64.msi `
-                 1597eef91b39fe4b34bab506158e34aa3a89490c519c97ac75a7c5d45885e345
+                 https://github.com/Kitware/CMake/releases/download/v3.18.4/cmake-3.18.4-win64-x64.msi `
+                 7e0cf19d4a1ea31de78487306c0126ec6326ab509d95e56dd4d604e9c0b34317
 RunAndCheckError "msiexec.exe" @("/i", "$env:TEMP\cmake.msi", "/quiet", "/norestart") $true
 AddToPath $env:ProgramFiles\CMake\bin
 
 # Ninja to ensure a 64-bit build of the tool (VS BuildTools ships a 32-bit build)
 mkdir -Force C:\tools\ninja
 DownloadAndCheck $env:TEMP\ninja.zip `
-                 https://github.com/ninja-build/ninja/releases/download/v1.10.0/ninja-win.zip `
-                 919fd158c16bf135e8a850bb4046ec1ce28a7439ee08b977cd0b7f6b3463d178
+                 https://github.com/ninja-build/ninja/releases/download/v1.10.1/ninja-win.zip `
+                 5d1211ea003ec9760ad7f5d313ebf0b659d4ffafa221187d2b4444bc03714a33
 Expand-Archive -Path $env:TEMP\ninja.zip -DestinationPath C:\tools\ninja
 AddToPath C:\tools\ninja
 
 # LLVM to ensure a 64-bit build of the tool (VS BuildTools ships a 32-bit build)
 DownloadAndCheck $env:TEMP\LLVM-win64.exe `
-                 https://github.com/llvm/llvm-project/releases/download/llvmorg-10.0.0/LLVM-10.0.0-win64.exe `
-                 893f8a12506f8ad29ca464d868fb432fdadd782786a10655b86575fc7fc1a562
+                 https://github.com/llvm/llvm-project/releases/download/llvmorg-11.0.0/LLVM-11.0.0-win64.exe `
+                 a773ee3519ecc8d68d91f0ec72ee939cbed8ded483ba8e10899dc19bccba1e22
```

Comment:
This will break clang-cl out of the box for developers. We haven't made any announcement of this, and llvm 10.0.0 had it's own list of optimization defects that didn't let us build a number of tests on Windows.

My suggestion is to ignore this breakage, it will be self-healing when the envoy WORKSPACE is updated to either bazel 3.7.1 or 4.0.0.

Reply:
Do you want to pull the LLVM and Bazel version changes out into a separate draft PR? We can bump the other deps standalone in this PR and get them working in the build quicker

Reply:
If this "works", I would rather have llvm 11.0.0 live. Bumping the WORKSPACE to bazel 4.0.0 will cause this to "just work" - I expected we would give this specific docker image a full spin, using the test build of bazel with the rules_cc bump.

Reply:
In any case, regenerating the toolchains now would be nice for testing bazel 3.7.0 ahead of the 4.0.0 build (across the board.)

Reply:
I think we saw that even the MSVC toolchain doesn't work given a Bazel version without our fix so this is unmergable as-is (the CI check to regenerate toolchains passes in this PR b/c of [this line](https://github.com/envoyproxy/envoy-build-tools/blob/5f3c8c6567f3e8c532ac4982d5284572f6e21d19/toolchains/regenerate.sh#L10-L13)), the CI check for post-merge and generate toolchains will fail

Reply:
I'm going to correct myself. We should leave the bazel change to 3.6.0+3.7.0 in place. We need to back off the llvm change until bazel 3.7.1/4.0.0. I'll adjust this now.

Even invoking bazel to build msvc-cl, building against llvm 11.0.0 still fails (this was @davinci26's earlier observation), see
https://github.com/envoyproxy/envoy/pull/13688#issuecomment-729231947